cmd/local.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"bytes"
+	"github.com/falcosecurity/driverkit/pkg/driverbuilder"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"golang.org/x/sys/unix"
+	"log/slog"
+	"os"
+	"os/user"
+	"runtime"
+	"strings"
+)
+
+type localCmdOptions struct {
+	useDKMS bool
+	srcDir  string
+	envMap  map[string]string
+}
+
+// NewLocalCmd creates the `driverkit local` command.
+func NewLocalCmd(rootCommand *RootCmd, rootOpts *RootOptions, rootFlags *pflag.FlagSet) *cobra.Command {
+	opts := localCmdOptions{}
+	localCmd := &cobra.Command{
+		Use:               "local",
+		Short:             "Build Falco kernel modules and eBPF probes in local env with local kernel sources and gcc/clang.",
+		PersistentPreRunE: persistentPreRunFunc(rootCommand, rootOpts),
+		Run: func(c *cobra.Command, args []string) {
+			slog.With("processor", c.Name()).Info("driver building, it will take a few seconds")
+			if !configOptions.DryRun {
```

Comment:
Just a stylistic nit: you could remove a level of indentation in this closure with 

```go
if configOptions.DryRun {
    return
}
```